Loader/Config/API.luau

```diff
@@ -1,5 +1,7 @@
 --[[
-	If you're learning how to create in Adonis, the wiki may be of help.
+	If you want to learn how to modify Adonis, create plugins and/or themes;
```

Comment:
This is difficult to understand and requires re-wording. I myself am not sure what to make of this.

Reply:
(lines 2-4)